news/2020-02-05-new-featured-artist-kola-kid.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: Kola Kid"
+date: 2020-02-05 09:00:00 +0000
+---
+
+**Kola Kid** bursts into the scene as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/67/header.jpg)
+
+Showcased foremost in [Bonsai](https://osu.ppy.sh/users/987334)'s [collab set](https://osu.ppy.sh/beatmapsets/489383#osu/1043092) for *press start* back in 2016, **Kola Kid** is the self-proclaimed Godfather of chiptune fusion. With bleeps and bloops inspired by the video game BGM of yesteryear mixed with tinges of modern electronica, breakcore, punk and rap, his work is a unique blend of a bunch of stuff that just straight up *works*.
+
+The Mappers' Guild has been hard at work creating a few sets to highlight exactly how the magic comes together in more modern mapping styles, piloted by the minds of [Mirash](https://osu.ppy.sh/users/2841009) and [ShirohaMyMommy](https://osu.ppy.sh/users/1603923) (aka: Delis).
+
+Sound interesting? Check out a few samples for yourself below:
+
+### press start
+
+*[Check out this collab set](https://osu.ppy.sh/beatmapsets/489383#osu/1043092)hosted by [Bonsai](https://osu.ppy.sh/users/987334)!*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1877.mp3" type="audio/mpeg">
+</audio>
+
+### timer
+
+*[Check out this set](https://osu.ppy.sh/beatmapsets/1089084#osu/2277126) by [Mirash](https://osu.ppy.sh/users/2841009)!*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1878.mp3" type="audio/mpeg">
+</audio>
+
+### can't hide your love
+
+*[Check out this set](https://osu.ppy.sh/beatmapsets/1099325#osu/2296606) by [ShirohaMyMommy](https://osu.ppy.sh/users/1603923)!*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1864.mp3" type="audio/mpeg">
+</audio>
+
+### contra88
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1870.mp3" type="audio/mpeg">
+</audio>
+
+### broken legs
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1869.mp3" type="audio/mpeg">
+</audio>
+
+----
+
+Like always, these tracks and a bunch more are available right now over at [**Kola Kid**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/67), all pre-timed and just waiting for you to press start. Head on over and get cracking!
+
+There were a lot of correct callouts this time around, especially as the astute among us are starting to notice some *definitely not suspicious* patterns emerging. Big-brained frickin' *radar* technically wins this week's callout by correctly guessing nearly a week before the hint actually was released.
```

Comment:
So who was this BN? that won?

Reply:
also, that's why mystery quests should be released at sparse times that make it seem like a usual ranking process

Reply:
radar....

Reply:
```suggestion
There were a lot of correct callouts this time around, especially as the astute among us are starting to notice some *definitely not suspicious* patterns emerging. Big-brained frickin' [https://osu.ppy.sh/users/7131099](radar) technically wins this week's callout by correctly guessing nearly a week before the hint actually was released.
```